CHANGES.txt

```diff
@@ -1,4 +1,8 @@
-2.4.0
+2.5.0 (2023-08-01)
+====================
+* Prioritize urllib.parse.quote import over deprecated django.utils.http.urlquote
```

Comment:
Also, it's good to note in the changelog here that this adds compatibility with Django 4.2.